usr/sbin/registercloudguest

```diff
@@ -484,6 +488,11 @@ prod_data = json.loads(res.text)
 extensions = prod_data.get('extensions')
 register_modules(extensions, products)
 
+if registration_returncode != 0:
+    print('Registration failed, see /var/log/guestregister for details')
```

Comment:
This is an error and needs to go to stderr, i.e. `file=sys.stderr'

exit code is set later in the code.